common/constants.go

```diff
@@ -680,6 +680,11 @@ const TriggerRegistryInitialKeyPrefix = "initial_value_epoch_"
 // NodesCoordinatorRegistryKeyPrefix is the key prefix to save epoch start registry to storage
 const NodesCoordinatorRegistryKeyPrefix = "indexHashed_"
 
+// EpochStartStaticBlocksKeyPrefix is the key prefix for epoch start meta block key
+const EpochStartStaticBlocksKeyPrefix = "epochStartBlock_"
```

Comment:
looks like it is not used

Reply:
right, renamed and removed the other variable